lsp-clients.el

```diff
@@ -806,6 +805,67 @@ responsiveness at the cost of possible stability issues."
                   :server-id 'emmy-lua
                   :notification-handlers (lsp-ht ("emmy/progressReport" #'ignore))))
 
+
+(defgroup lsp-lua-language-server nil
```

Comment:
can you add :version "7.1" for all newly introduced properties?

Reply:
done!